rasa_core/training/structures.py

```diff
@@ -34,6 +37,23 @@
 
 GENERATED_HASH_LENGTH = 5
 
+FORM_PREFIX = "form: "
+
+
+class AsStoryStringHelper(object):
+    def __init__(self,
+                 active_form=None,
+                 form_validation=True,
+                 form_rejected=False,
+                 form_string='',
+                 no_form_string=''):
+
+        self.active_form = active_form
+        self.form_validation = form_validation
+        self.form_rejected = form_rejected
+        self.form_string = form_string
+        self.no_form_string = no_form_string
```

Comment:
wtf is this `no_form_string' hahaha

Reply:
i guess something that's not a form? make that more obvious please

Reply:
ok maybe `form_prefix_string` and `no_form_prefix_string`

Reply:
👍